transcom/mymove
# MB-15873 - Add office sit address updates endpoint data

## [Jira ticket](https://dp3.atlassian.net/browse/MB-15873)

## Summary

This PR creates a swagger def for SITAddressUpdates and adds related data to the payload of the ListMTOServiceItems endpoint.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Run the testharness scenario `HHGMoveInSITWithAddressChangeRequestOver50Miles` and copy the move code
2. Access the office app as a TOO
3. Navigate to the move's MTO tab
4. Check the mto_service_items payload and validate that the data returned for SITAddressUpdates is correct

### Frontend


### Backend
